packages/react/src/isomorphicClerk.ts

```diff
@@ -137,6 +137,16 @@ type IsomorphicLoadedClerk = Without<
   client: ClientResource | undefined;
 };
 
+class ClerkMap extends Map {
+  name = 'clerk-react';
+
+  constructor() {
+    super();
+  }
+}
+
+const cache = new ClerkMap();
```

Comment:
If we're gonna have this Map wrapper, maybe we can define it in `shared`?

Reply:
i will drop this, as i mentioned above